### PR TITLE
quandl for financial data

### PR DESCRIPTION
**Pull request by alen12345**

### CODE REVIEW

File: recipes/quandl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+test:
+  imports:
+    - test
+
+about:
+  home: https://www.quandl.com/
+  license: MIT
+  license_family: MIT
+  # have requested to add it to the upstream
+  # license_file: LICENSE.txt
```
Comment (CJ-Wright) [inline recipes/quandl/meta.yaml]: Can you please package the license?

Comment (alen12345) [reply recipes/quandl/meta.yaml]: There is no license in the upstream; it is not even in the github source. Can you please suggest me what to do?

Comment (CJ-Wright) [reply recipes/quandl/meta.yaml]: Then you are good to go I guess. (although you could include a link to the issue asking for a license)

Comment (alen12345) [reply recipes/quandl/meta.yaml]: Let me do that and then if you think we are ready you can merge ;)

**Comment by alen12345** [comment]
@CJ-Wright added link to the opened issue and all tests passed :) when you have time, we should be good to go!

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**